pyodk/endpoints/projects.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from dataclasses import dataclass, fields
+from datetime import datetime
+from typing import Dict, List, Optional
+
+from pyodk import validators
+from pyodk.endpoints.utils import error_if_not_200
+from pyodk.errors import PyODKError
+from pyodk.session import ClientSession
+from pyodk.utils import STRPTIME_FMT_UTC
+
+log = logging.getLogger(__name__)
+
+
+@dataclass
+class ProjectEntity:
```

Comment:
Given that we're introducing an Entity concept to ODK I think it would be ideal to avoid the `Entity` suffix. Could it just be Project, Form, etc?

Reply:
Maybe `ProjectMetadata`, etc?

Reply:
I've renamed the data model classes to the plain names `Project`, `Form`, `Submission`. Please see updated OP comment for details on other enhancements in the latest commit d342233.